pgo/cmd/cluster.go

```diff
@@ -230,7 +235,9 @@ func createCluster(args []string, ns string, createClusterCmd *cobra.Command) {
 	r.Namespace = ns
 	r.ReplicaCount = ClusterReplicaCount
 	r.NodeLabel = NodeLabel
-	r.Password = Password
+	r.PasswordSuperuser = PasswordSuperuser
+	r.PasswordReplication = PasswordReplication
+	r.PasswordUser = PasswordUser
```

Comment:
Per earlier comment, I think this should remain `Password` to avoid API breakage

Reply:
Changed back to `Password`.